renderdoc/driver/vulkan/wrappers/vk_queue_funcs.cpp

```diff
@@ -1043,6 +1043,9 @@ void WrappedVulkan::CaptureQueueSubmit(VkQueue queue,
 
       record->bakedCommands->AddRef();
     }
+
+    if(backframe || capframe)
```

Comment:
nit: This check is redundant as we're always in either one of these states during capture.